features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -238,7 +237,6 @@ internal class DefaultImageWireframeHelper(
                 }
             }
 
-            is GradientDrawable -> DrawableProperties(drawable, view.width, view.height)
```

Comment:
why this PR has this removed?

Reply:
I will elaborate what is written in description:
"Fix the issue that GradientDrawable didn't take into account the width&height after resizing."
So after fixing the padding issue, I found that `GradientDrawable` is not correct resized,  which makes the views (purple circles and squares) exceed their bounds, so I have to fix it by removing this line. By removing this line, `GradientDrawable` will not be handled differently, it will fallback to `else` condition.